org.keepassxc.KeePassXC.json

```diff
@@ -0,0 +1,123 @@
+{
+  "id": "org.keepassxc.KeePassXC",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "keepassxc",
+  "rename-icon": "keepassxc",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc", "--socket=x11",
+    /* Notification access */
+    "--talk-name=org.freedesktop.Notifications",
+    /* Screen Lock Listener */
+    "--talk-name=org.freedesktop.login1.Manager",
+    "--talk-name=org.freedesktop.ScreenSaver",
+    "--talk-name=org.gnome.SessionManager",
+    "--talk-name=org.gnome.SessionManager.Presence",
+    "--talk-name=com.canonical.Unity.Session",
+    /* KeePassHTTP */
+    "--share=network",
+    /* YubiKey USB access */
+    "--device=all"
+  ],
+  "cleanup": [
+    "*.a",
+    "*.h",
+    "*.la",
+    "/include",
+    "/lib/pkgconfig"
+  ],
+  "modules": [
+    {
+      "name": "keepassxc",
+      "buildsystem": "cmake",
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DWITH_TESTS=OFF",
```

Comment:
Wouldn't it be better to enable tests?

Reply:
The decision to turn of tests was based upon a quick overview of upstream packaging. For whatever reason they build their official snap and AppImage with tests turned off.

I searched their GitHub project and checked the history on their build recipes but failed to find any reason as to why they do this.

Despite this the `CMakeList.txt` still has it on by default. So I removed `-DWITH_TESTS=OFF` and the tests seem to run without any issue. I am no expert in this area though.

Since there are no issues with the tests and I cannot imagine any significant difference in build time, I will use the default setting (ON).